OneSignalSDK/unittest/src/test/java/com/test/onesignal/JsonAsserts.java

```diff
@@ -24,10 +24,13 @@ static void containsSubset(@NonNull JSONObject subject, @NonNull JSONObject cont
       Iterator<String> iterator = contains.keys();
       while (iterator.hasNext()) {
          String key = iterator.next();
-         assertEquals(
-            normalizeType(contains.get(key)),
-            normalizeType(subject.get(key))
-         );
+         if (contains.get(key) instanceof JSONObject && subject.get(key) instanceof JSONObject)
+            equals((JSONObject) contains.get(key), (JSONObject) subject.get(key));
```

Comment:
This won't work, because getJSONObject fails and it will make all the assert to fail, I'm extending this method to check for jsonObject equals inside containsSubset

Reply:
Sorry I just mean with the `equals` on line 28. 
```java
 equals(contains.getJSONObject(key), subject.getJSONObject(key));
```

Reply:
oh gotcha!